### ISSUE TITLE
Is it possible to dim down the colors of the HTML comments?

### ISSUE THREAD

**Issue by nextgenthemes** [co6x0/vscode-wp-block-html#1]
Thanks for the extension.

I like the highlighting, but I would prefer the comments to be more toned down compared to the HTML. I know nothing about VS Code extensions, and just checking if you know if that is possible at all.

**Comment by co6x0** [comment]
Thank you for reporting.

In my environment, the extension highlights as shown in the screenshot below.
I am using [Atom One Dark theme](https://marketplace.visualstudio.com/items?itemName=akamud.vscode-theme-onedark).

<img width="491" alt="CleanShot 2024-03-02 at 08 47 08@2x" src="https://github.com/co6x0/vscode-wp-block-html/assets/34850483/c9d75618-9457-43f9-a34d-ffcb4c060cc3">

Maybe this extension does not work well with your VS Code theme.
Can you let me know what theme you are using?

I will try to find out how this extension works with that theme.

**Comment by nextgenthemes** [comment]
I mean the `<-- wp:block -->` comments. I would like them to be colored, just not as bright/saturated colors as the HTML.

I just wanted to know if you know is that is possible. Like, a basic CSS way would just be to give the elements a lowered opacity. More advanced would be to take the original colors and reduce saturation/brightness.

**Comment by co6x0** [comment]
I see.
You want to change the color of the syntax highlight applied to the Block Markup?

Unfortunately that is not as easy to change as CSS.

Block Markup is a unique WordPress syntax, so VS Code cannot color it as HTML syntax. (That's just an HTML comment.)
Therefore, `vscode-wp-block-html` teaches editors to interpret Block Markup as standard HTML syntax, so that syntax highlighting will work in the various VS Code themes.

In other words, I think it is currently difficult to set a special color just for Block Markup.

I am not too familiar with this, but perhaps you can take advantage of TextMate grammars to customize the theme to your liking.
<https://code.visualstudio.com/docs/getstarted/themes#_editor-syntax-highlighting>

**Comment by nextgenthemes** [comment]
Thanks for showing me a possible direction. I can't find the time to look into it, and it's not that important after all.
